LuaMenu/widgets/gui_draw_everywhere.lua

```diff
@@ -14,4 +14,3 @@ function widget:DrawScreen()
 	gl.Color(1,0,0.8,1)
 	gl.Rect(-10, -10, 5000, 5000)
 end
-	
```

Comment:
Why is there no newline at the end of this file?

Reply:
It is there on my PC, odd.. I added an extra one now